requirements.txt

```diff
@@ -27,4 +27,5 @@ typing_extensions<4.6.0
 pyjwt~=2.6.0
 websockets~=12.0.0
 pandas
-xlsxwriter~=3.2.0
\ No newline at end of file
+xlsxwriter~=3.2.0
+xmltodict
```

Comment:
Let's also add this in the [setup.cfg](https://github.com/mbrg/power-pwn/blob/main/setup.cfg) file.

Reply:
done